.github/workflows/_run-pact-provider-tests.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Pull docker image dependencies
         run: |
-          docker pull govukpay/postgres:11.1
+          docker pull postgres:11.16
```

Comment:
Is this container change related to M1 support? If it is, it'd be good to write an explanation in the commit message.